src/omv/ports/mimxrt/omv_spi.c

```diff
@@ -78,65 +107,118 @@ static void spi_master_callback(LPSPI_Type *base, void *handle, status_t status,
     } else {
         spi->xfer_flags |= OMV_SPI_XFER_FAILED;
         spi->xfer_error = status;
-        if (spi->xfer_flags & OMV_SPI_XFER_DMA) {
-            LPSPI_MasterTransferAbortEDMA(spi->inst, &spi->descr_master_edma);
+        omv_spi_transfer_abort(spi);
+    }
+
+    void *rxBuf = spi->xfer_descr.rxData;
```

Comment:
I think this should be `rxData` or `txData` otherwise it will be NULL in TX transfer callback, even if it's not used by the callback it shouldn't be null.

Reply:
My intention is that it would be null unless it was an rx callback. I can change that though.

Reply:
Done. Will return a pointer to the transmit buffer if no rx buffer is present.